pkg/chains/signing.go

```diff
@@ -204,8 +214,9 @@ func (o *ObjectSigner) Sign(ctx context.Context, tektonObj objects.TektonObject)
 					merr = multierror.Append(merr, err)
 				} else {
 					logger.Infof("Uploaded entry to %s with index %d", cfg.Transparency.URL, *entry.LogIndex)
-
 					extraAnnotations[ChainsTransparencyAnnotation] = fmt.Sprintf("%s/api/v1/log/entries?logIndex=%d", cfg.Transparency.URL, *entry.LogIndex)
+					logger.Debugf("Recording count of uploaded artifacts for %v", obj)
```

Comment:
Trace level?

Reply:
Removed the loggers as trace level  is not available.